xExtension-YouTube/extension.php

```diff
@@ -213,7 +274,7 @@ public function handleConfigureAction()
         if (Minz_Request::isPost()) {
             FreshRSS_Context::$user_conf->yt_player_height = (int)Minz_Request::param('yt_height', '');
             FreshRSS_Context::$user_conf->yt_player_width = (int)Minz_Request::param('yt_width', '');
-            FreshRSS_Context::$user_conf->yt_show_content = (int)Minz_Request::param('yt_show_content', 0);
+            FreshRSS_Context::$user_conf->yt_content_handling = (string)Minz_Request::param('yt_content_choice', '');
```

Comment:
Maybe the default value (second parameter of `param`) should be `none` or other default value of `contentHandling`.

(nitpick) You don't need to cast a string in string ;)

Reply:
Ah yes, indeed, I wasn't sure of the actual return type.

For the default, my idea was to delegate that to settings loading in `loadConfigValues()`, so it's all in the same place. A bit like for height & width.

Reply:
Ok then :+1: